lib/rspec/support.rb

```diff
@@ -1,5 +1,13 @@
 # frozen_string_literal: true
 
+class Thread
+  attr_accessor :__rspec_local_data
+
+  class << self
+    alias __current_for_rspec current
```

Comment:
I don't have a strong opinion about it but I always try to have a "namespace" first, e.g. `__rspec_current` (`__rspec_whatever`).